kong/db/schema/typedefs.lua

```diff
@@ -252,4 +252,11 @@ typedefs.key = Schema.define {
 }
 
 
+setmetatable(typedefs, {
+  __index = function(_, k)
+    error("schema typedef error: definition " .. k .. " does not exist")
```

Comment:
I suggest using a stack level of 2 here:
```lua
error("...", 2)
```

This way the plugin author will easily track down the issue in their code.

Reply:
Great call!

Reply:
Updated